web/satellite/src/api/payments.ts

```diff
@@ -193,4 +195,24 @@ export class PaymentsHttpApi implements PaymentsApi {
 
         return [];
     }
+
+    /**
+     * Process coin payments
+     * @param amount
+     * @throws Error
+     */
+    public async makeTokenDeposit(amount: string): Promise<DepositInfo> {
+        const path = `${this.ROOT_PATH}/tokens/deposit`;
+        const response = await this.client.post(path, JSON.stringify({amount}));
+
+        if (!response.ok) {
+            if (response.status === 401) {
+                throw new ErrorUnauthorized();
+            }
+
+            throw new Error('can not process coin payment');
```

Comment:
api response could contain 3 error codes:
401 - unauthrized
400 - bad data
500 - internal error

we could possibly add new error check to improve err message